extensions-core/lookups-cached-single/src/main/java/org/apache/druid/server/lookup/PollingLookup.java

```diff
@@ -132,9 +132,7 @@ public String apply(@Nullable String key)
       return NullHandling.emptyToNullIfNeeded((String) cache.get(keyEquivalent));
     }
     finally {
-      if (cacheRefKeeper != null && cache != null) {
```

Comment:
I believe `cache != null` is necessary

Reply:
it seems the code block is not dependent on cache, but I have included the `cache != null` check